assets/js/index.js

```diff
@@ -0,0 +1,249 @@
+const API_KEY = "b248c5c1ee789732d38aa9fe65ef7935";
+const form = document.getElementById("form");
```

Comment:
You've used a mixture of jQuery and standard jacascript through your code - please try to keep it consistent. In this case, I would expect to see jQuery used throughout - here this would look like:

`const form = $("#form")`